acceptance/openstack/sharedfilesystems/v2/shares_test.go

```diff
@@ -353,6 +353,73 @@ func TestRevert(t *testing.T) {
 	t.Logf("Share %s successfuly reverted", share.ID)
 }
 
+func TestShareRestoreFromSnapshot(t *testing.T) {
+	client, err := clients.NewSharedFileSystemV2Client()
+	if err != nil {
+		t.Fatalf("Unable to create a shared file system client: %v", err)
+	}
+	client.Microversion = "2.27"
```

Comment:
The microversion doesn't seem right, [2.27 is where revert to snapshot was added](https://docs.openstack.org/manila/latest/contributor/api_microversion_history.html#id25), and it looks like that's not what we're using here. Not a big deal really, since we're way past microversion 2.27 (Ocata) anyway.

Reply:
but agree with the point here that "restoring from snapshot" here is really "creating share from snapshot" in manila terminology; and confusingly, manila has a feature called "revert_to_snapshot" which allows in-place reverts.. and this "2.27" version seems to come from that.